<!DOCTYPE html>
<html lang='en'>
<head>
<title>0001-enable-linux-unstable-deb-target.patch - aur.git - AUR Package Repositories</title>
<meta name='generator' content='cgit v1.2.3-56-gcc08'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/static/css/cgit.css'/>
<script type='text/javascript' src='/cgit.js'></script>
<link rel='shortcut icon' href='/static/images/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='https://aur.archlinux.org/cgit/aur.git/atom/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi' type='application/atom+xml'/>
<link rel='vcs-git' href='https://aur.archlinux.org/chromium-wayland-vaapi.git/' title='aur.git Git repository'/>
</head>
<body>
		<div id="archnavbar" class="anb-aur">
			<div id="archnavbarlogo"><h1><a href="/" title="Return to the main page">Arch Linux User Repository</a></h1></div>
			<div id="archnavbarmenu">
				<ul id="archnavbarlist">
					<li id="anb-home"><a href="https://www.archlinux.org/" title="Arch news, packages, projects and more">Home</a></li>
					<li id="anb-packages"><a href="https://www.archlinux.org/packages/" title="Arch Package Database">Packages</a></li>
					<li id="anb-forums"><a href="https://bbs.archlinux.org/" title="Community forums">Forums</a></li>
					<li id="anb-wiki"><a href="https://wiki.archlinux.org/" title="Community documentation">Wiki</a></li>
					<li id="anb-gitlab"><a href="https://gitlab.archlinux.org/archlinux" title="GitLab">GitLab</a></li>
					<li id="anb-security"><a href="https://security.archlinux.org/" title="Arch Linux Security Tracker">Security</a></li>
					<li id="anb-aur"><a href="/" title="Arch Linux User Repository">AUR</a></li>
					<li id="anb-download"><a href="https://www.archlinux.org/download/" title="Get Arch Linux">Download</a></li>
				</ul>
			</div>
		</div><!-- #archnavbar -->
<div id='cgit'><table id='header'>
<tr>
<td class='main'>AUR : chromium-wayland-vaapi.git</td></tr>
<tr><td class='sub'>AUR Package Repositories | click <a href='/pkgbase/chromium-wayland-vaapi/'>here</a> to return to the package base details page</td><td class='sub right'></td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/aur.git/?h=chromium-wayland-vaapi'>summary</a><a href='/cgit/aur.git/log/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>log</a><a class='active' href='/cgit/aur.git/tree/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>tree</a><a href='/cgit/aur.git/commit/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>commit</a><a href='/cgit/aur.git/diff/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>diff</a><a href='/cgit/aur.git/stats/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/aur.git/log/0001-enable-linux-unstable-deb-target.patch'>
<input type='hidden' name='h' value='chromium-wayland-vaapi'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='search' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/aur.git/tree/?h=chromium-wayland-vaapi'>root</a>/<a href='/cgit/aur.git/tree/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>0001-enable-linux-unstable-deb-target.patch</a></div><div class='content'>blob: 4ae23e0ca07564cf30058c0b427bdddc5967250d (<a href='/cgit/aur.git/plain/0001-enable-linux-unstable-deb-target.patch?h=chromium-wayland-vaapi'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
</pre></td>
<td class='lines'><pre><code>From 22fe045deac0551720d7292022f443b22703f336 Mon Sep 17 00:00:00 2001
From: Yaowei Zhou &lt;yaowei.zhou&#64;intel.com&gt;
Date: Thu, 20 Apr 2023 14:37:26 +0800
Subject: [PATCH] Enable &quot;linux:unstable_deb&quot; build target when using the chromium minigbm

Bug: N/A
Change-Id: Ic37059be0f0719895acdba756292aed71820feba
<span class="hl kwa">---</span>

diff --git a/third_party/minigbm/BUILD.gn b/third_party/minigbm/BUILD.gn
index 739e278..50bf5d8 100644
<span class="hl kwa">--- a/third_party/minigbm/BUILD.gn</span>
<span class="hl kwb">+++ b/third_party/minigbm/BUILD.gn</span>
<span class="hl kwd">&#64;&#64; -58,7 +58,7 &#64;&#64;</span>
     }
   }
 
<span class="hl kwa">-  shared_library(&quot;minigbm&quot;) {</span>
<span class="hl kwb">+  static_library(&quot;minigbm&quot;) {</span>
     sources = [
       &quot;src/amdgpu.c&quot;,
       &quot;src/dri.c&quot;,
<span class="hl kwd">&#64;&#64; -89,8 +89,8 &#64;&#64;</span>
     }
 
     # Clients need this to pick up the shared library correctly.
<span class="hl kwa">-    all_dependent_configs =</span>
<span class="hl kwa">-        [ &quot;//build/config/gcc:rpath_for_built_shared_libraries&quot; ]</span>
<span class="hl kwb">+    #all_dependent_configs =</span>
<span class="hl kwb">+    #    [ &quot;//build/config/gcc:rpath_for_built_shared_libraries&quot; ]</span>
   }
 
   # This target is used for Chromecast build, which expects the resulting lib
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div id="footer">
	<p>
		Copyright &copy; 2004-2023 aurweb Development Team &ndash;
		<strong>AUR packages are user produced content. Any use of the provided files is at your own risk.</strong>
	</p>
</div>
</div> <!-- id=cgit -->
</body>
</html>
